src/utils.js

```diff
@@ -0,0 +1,17 @@
+import yaml from 'js-yaml';
+
+// extracts yaml front matter from a markdown file path
+export default function frontMatterParser(content) {
+  // format file to extract yaml front matter
+  const results = content.split('---');
+  const articleConfig = results[1];
+  const articleContent = results[2];
+
+  // get the configs
+  const configObj = yaml.safeLoad(articleConfig);
+
```

Comment:
Maybe we could rename the return values as `frontMatter` and `mdBody`?

Reply:
Renamed as frontMatter and mdBody